integration/mediation-tests/tests-patches/src/test/java/org/wso2/carbon/esb/passthru/transport/test/ForceMessageValidationTestCase.java

```diff
@@ -59,9 +59,10 @@ public void setEnvironment() throws Exception {
         serverConfigurationManager.applyConfiguration(new File(
                 getESBResourceLocation().replace("//", "/") + File.separator + "passthru" + File.separator + "transport"
                         + File.separator + "forceMessageValidation" + File.separator + "passthru-http.properties"));
+        CarbonServerExtension.restartServer();
```

Comment:
Shall we look into the possibility of applying configuration and doing an optional restart there itself writing a new method as we discussed offline.

Reply:
Done the changes. Please review.